app/views/pages/service_start.cy.html.erb

```diff
@@ -0,0 +1,231 @@
+<%= content_for :additional_links do %>
+  <%= javascript_pack_tag 'navigation' %>
+<%- end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l">How to apply for access to NPD data</h1>
+    <p>Cymraeg version coming soon</p>
+    <hr class="govuk-section-break govuk-section-break--xl  govuk-section-break--visible ">
+  </div>
+
+  <div class="govuk-grid-column-two-thirds">
+    <div>
+      <p class="govuk-!-font-size-16">
+        Published <%= l(Date.today, format: :npd) %><br />
+        Last updated <%= l(Date.today, format: :npd) %> - <a href="">see all updates</a><br />
```

Comment:
Is this valid? The page wasn't necessarily updated or published on the day it was updated. It would be preferable to remove components like this where we don't have it wired up, so the live version is "correct" even though it's more barebones

Reply:
You're right, I will remove it from both versions.